adafruit_apds9960/apds9960.py

```diff
@@ -169,6 +177,17 @@ def _reset_counts(self):
     """Proximity interrupt enable flag.  True if enabled,
         False to disable"""
 
+    ## GESTURE ROTATION
+    @property
+    def rotation(self):
+        """Gesture rotation offset"""
+        return self._rotation
+
+    @rotation.setter
+    def rotation(self, new_rotation):
+        if new_rotation in [0, 90, 180, 270]:
+            self._rotation = new_rotation
+
```

Comment:
I think this should throw a ValueError if the input is not in [0,90,180,270] - it will just ignore it silently as written

Reply:
Thanks